styles.css

```diff
@@ -6,419 +6,1824 @@
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #f4f1f0;
+  }
+
+  .btn-settings:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  .resume:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  footer {
+    display: flex;
+    justify-content: center;
+  }
+
+  .border {
+    border: 1px solid #dfe1e6;
+    margin-bottom: 8px;
+  }
+
+  #foots {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .footer-icons {
+    display: flex;
+    gap: 24px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+  }
+
+  .footer-icons li img:hover {
+    width: 15px;
+    height: 15px;
+  }
+
+  .desktpopupcontainer {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.85);
+    display: none;
+    backdrop-filter: blur (30px);
+  }
+
+  .expanded {
+    display: block;
+  }
+
+  .desktmodalwindowcontainer {
+    background-color: #fff;
+    margin: 5% auto;
+    padding: 16px;
+    height: 80%;
+    width: 70%;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  .desktpopup-header {
+    display: flex;
+    padding-bottom: 2%;
+  }
+
+  .deskttrackkeeping {
+    width: 259px;
+    height: 82px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .desktclose-button {
+    font-size: 40px;
+    color: #67798e;
+    transform: rotate(45deg);
+    position: absolute;
+    top: 5rem;
+    right: 250px;
+    cursor: pointer;
+    transition: font-size ease-in-out 0.75s;
+  }
+
+  .Ddesktopnewlang {
+    padding-bottom: 2%;
+    display: flex;
+    gap: 19px;
+  }
+
+  .DdesknewLang {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+    gap: 24px;
+    width: 100px;
+    left: calc(50% - 343px / 2);
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    color: #3c3a39;
+  }
+
+  .desktimg-popup {
+    width: 550px;
+    height: 360px;
+    top: 156px;
+  }
+
+  .desktpicturefttext {
+    display: flex;
+    gap: 23px;
+  }
+
+  .desktthetxt {
+    width: 424px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+    letter-spacing: 1px;
+  }
+
+  .desktlivedisplay {
+    display: flex;
+    gap: 19px;
+    margin-top: 70px;
+  }
+
+  .desktcardbutton1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    padding: 12px;
+    height: 48px;
+    width: 134px;
+    background: #ff6b00;
+    border: none;
+    gap: 3px;
+  }
```

Comment:
- [ ] Great job 👏 adding the proper button styling here but it seems like you forgot to implement the `:hover` and `:active` states as specified in the Figma design, so kindly add that:
- Figma design:
    ![firefox_PKDzK8XvCT](https://user-images.githubusercontent.com/104968236/232014942-535379e9-7908-4b20-a22a-36291ce9f171.png)